cephfs/path_test.go

```diff
@@ -60,3 +60,273 @@ func TestRemoveDir(t *testing.T) {
 	assert.EqualError(t, err,
 		fmt.Sprintf("stat %s: no such file or directory", localPath))
 }
+
+func TestLink(t *testing.T) {
+	mount := fsConnect(t)
+	defer fsDisconnect(t, mount)
+
+	t.Run("directory operations", func(t *testing.T) {
+		// Root dir, both as source and destination.
+		err := mount.Link("/", "/")
+		// Error directory operations are not allowed.
+		assert.Error(t, err)
+
+		dirname := "mydir"
+		localPath := path.Join(CephMountDir, dirname)
+		err = mount.MakeDir(dirname, 0755)
+		assert.NoError(t, err)
+		_, err = os.Stat(localPath)
```

Comment:
If you use os functions in a test case please also call `useMount(t)` before making use of the local mount point. This test ensures the environment is set up to use the local cephfs mount and skips or fails the test case appropriately (see PR #204 for more context).
You may also just skip this stat if you wish, as I do not see it adding much extra value after we check that MakeDir above didn't return an error.

Reply:
Done